src/models/AppState.js

```diff
@@ -481,6 +481,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return policy && policy.rememberDeviceByDefault;
         }
+      },
+      'unauthenticatedToken': {
+        deps: ['transaction'],
+        fn: function (transaction) {
+          if (transaction && transaction.status === 'UNAUTHENTICATED' && transaction.data) {
```

Comment:
What's the actual flow on the Okta side? In general, the sign-in-widget shouldn't know about the stateToken, which is handled by AuthJs. In the flows where we bootstrap with a state (i.e. by passing a stateToken into the widget), we pass it to AuthJs via a resume call, but at that point it's loaded into AuthJs and will be attached to any subsequent request. 

If AuthJs is not passing a saved stateToken in the primaryAuth call, it feels like this fix should go there instead (rather than passing it in via the widget).